package.json

```diff
@@ -11,7 +11,8 @@
     "url": "https://github.com/dbader/node-datadog-metrics/issues"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha --reporter spec && ./node_modules/.bin/jshint --reporter node_modules/jshint-stylish-ex/stylish.js *.js ./lib/*.js ./test/*.js && ./node_modules/.bin/jscs --config .jscsrc *.js ./lib/*.js ./test/*.js"
+    "test": "./node_modules/.bin/mocha --reporter spec",
+    "check-codestyle": "./node_modules/.bin/jshint --reporter node_modules/jshint-stylish-ex/stylish.js *.js ./lib/*.js ./test/*.js && ./node_modules/.bin/jscs --config .jscsrc *.js ./lib/*.js ./test/*.js"
```

Comment:
Prefer `mocha`, `jshint`, and `jscs` over `./node_modules/.bin/...`

Reply:
Fixed 👍